### PR TITLE
'Demo' app blog post

### PR DESCRIPTION
**Pull request by will-moore**
Blog post about the OMERO.figure demo app.

### CODE REVIEW

File: _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<div class="panel panel-default">
+  <div class="panel-body" style="padding:0; text-align: center">
+    <img src="{{ site.baseurl }}/images/blog-omero-figure.png" style="width:600px"/>
+  </div>
+  <div class="panel-heading">
+    <h3 class="panel-title">OMERO.figure reads image data from OMERO, and can save figure files and PDFs</h3>
+  </div>
+</div>
+
+However, OMERO.figure doesn't **have** to get data from it's own "host" OMERO server. It
```
Comment (ximenesuk) [inline _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown]: it's -> its

Comment (ximenesuk) [inline _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown]: it's -> its

File: _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+<div class="panel panel-default">
+  <div class="panel-body" style="padding:0; text-align: center">
+    <img src="{{ site.baseurl }}/images/blog-demo-figure.png" style="width:600px"/>
+  </div>
+  <div class="panel-heading">
+    <h3 class="panel-title">The demo app reads image data from the OMERO-based JCB dataviewer</h3>
+  </div>
+</div>
+
+The demo app doesn't have it's own OMERO server. The html and javascript simply come from the
```
Comment (ximenesuk) [inline _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown]: it's -> its

File: _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  <div class="panel-body" style="padding:0; text-align: center">
+    <img src="{{ site.baseurl }}/images/blog-omero-figure.png" style="width:600px"/>
+  </div>
+  <div class="panel-heading">
+    <h3 class="panel-title">OMERO.figure reads image data from OMERO, and can save figure files and PDFs</h3>
+  </div>
+</div>
+
+However, OMERO.figure doesn't **have** to get data from it's own "host" OMERO server. It
+can get data from any OMERO server that is available on the internet.
```
Comment (ximenesuk) [inline _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown]: Is it worth qualifying this to clarify that the user needs access or the server has public access?

File: _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  <div class="panel-body" style="padding:0; text-align: center">
+    <img src="{{ site.baseurl }}/images/blog-demo-figure.png" style="width:600px"/>
+  </div>
+  <div class="panel-heading">
+    <h3 class="panel-title">The demo app reads image data from the OMERO-based JCB dataviewer</h3>
+  </div>
+</div>
+
+The demo app doesn't have it's own OMERO server. The html and javascript simply come from the 
+figure.openmicroscopy web site (which is hosted by github). The demo app instead
```
Comment (ximenesuk) [inline _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown]: github -> GitHub (provide a link too?)

File: _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+gets it's OMERO images and metadata from the [JCB dataviewer](http://jcb-dataviewer.rupress.org/), which is a public
```
Comment (ximenesuk) [inline _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown]: it's -> its

File: _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+        <img src="{{ site.baseurl }}/images/demo_add_images.png" style="width:600px"/>
+	</a>
+  </div>
+  <div class="panel-heading">
+    <h3 class="panel-title">Adding images from JCB dataviewer to OMERO.figure demo app</h3>
+  </div>
+</div>
+
+In fact, it's not just the demo app that can add images from publicly available OMERO servers.
+The regular OMERO.figure also has this feature (in case you want to do that?!).
```
Comment (ximenesuk) [inline _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown]: I'd drop the ?! at the end, it's not too weird!

**Comment by ximenesuk** [comment]
It reads okay though I haven't looked at it with the images rendered in situ.

File: _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+    <h3 class="panel-title">The demo app reads image data from the OMERO-based JCB dataviewer</h3>
+  </div>
+</div>
+
+The demo app doesn't have it's own OMERO server. The html and javascript simply come from the 
+figure.openmicroscopy web site (which is hosted by github). The demo app instead
+gets it's OMERO images and metadata from the [JCB dataviewer](http://jcb-dataviewer.rupress.org/), which is a public
+image repository built with OMERO.
+
+The JCB dataviewer provides public image data and image rendering, but doesn't support saving
```
Comment (mtbc) [inline _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown]: DataViewer

**Comment by hflynn** [comment]
Now staged at http://figure.staging.openmicroscopy.org/blog/

**Comment by will-moore** [comment]
NB: Info for how to stage gh-pages PRs is at https://www.openmicroscopy.org/site/support/contributing/ci-docs.html#term-figure-help-staging

File: _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown (hunk 4 above)
Comment (will-moore) [inline _posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown]: I think I'll just remove the mention of github - Just confusing for most users and not needed.

**Comment by will-moore** [comment]
All issues fixed. Any more comments? If not, will merge later. Thanks all.

**Comment by ximenesuk** [comment]
The fixes look good to me.

**Pull request merged**